alacritty.yml

```diff
@@ -337,16 +337,16 @@
 # directory of the parent process will be used.
 #working_directory: None
 
-# Windows 10 ConPTY backend (Windows only)
+# Windows 10 WinPTY backend (Windows only)
```

Comment:
WinPTY is not about `Windows 10` anymore, technically it's also available on previous Windows versions and enabling this option there might make sense for users (fallback works, but they'd eliminate that codepath).